config/locales/en.yml

```diff
@@ -22,7 +22,7 @@ en:
     links: Links
     topics: Topics
   signs:
-    sign_of_the_day: Show me a sign
+    sign_of_the_day: Sign of the day
```

Comment:
Is this a UI change?

Reply:
It is, I have provisional approval from @DSRU for it - this change will need to live in staging long enough for them to test it there.

Reply:
@eoinkelly how long do you suggest we keep it on staging?

Reply:
I wanted you to have time to test the feature - we can move it to prod if you are happy

Reply:
OK let's leave it at least a day then so that I can see how the caching works out.